### PR TITLE
sqldef: init at 0.12.7

### PR DESCRIPTION
**Pull request by kgtkr**
###### Motivation for this change
add [sqldef](https://github.com/k0kubun/sqldef)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/sqldef/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  src = fetchFromGitHub {
+    owner = "k0kubun";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-T8BhGO860G8mlvVzspR2XbHsWrVWYmsTW+tc86Bskr4=";
+  };
+
+  vendorSha256 = "sha256-f7f7VGWu8PDZhb5FlqJRfR9zPutpmN3UYuKLxvh4RDE=";
+
+  doCheck = false;
```
Comment (applePrincess) [inline pkgs/development/tools/sqldef/default.nix:16]: Comment the reason why we skip tests.

File: pkgs/development/tools/sqldef/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  meta = with lib; {
+    description = "Idempotent SQL schema management tool";
+    license = licenses.mit;
```
Comment (applePrincess) [inline pkgs/development/tools/sqldef/default.nix:20]: This package has two licenses, for parser and for everything else. So license should be
```suggestion
    license = with licenses; [ mit /* for everythnig except parser */  asl20 /* for parser */ ];
```

File: pkgs/development/tools/sqldef/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    homepage = "https://github.com/k0kubun/sqldef";
+    maintainers = with maintainers; [ kgtkr ];
+    platforms = platforms.unix;
```
Comment (applePrincess) [inline pkgs/development/tools/sqldef/default.nix:23]: nix defaults to platforms.unix so this line is redundant.

**Comment by kgtkr** [comment]
@applePrincess
Thank you for reviewing.
The fix is done.

**Comment by onny** [comment]
Could you please change commit and PR title to `sqldef: init at 0.12.7`. Also please squash commits

**Comment by applePrincess** [comment]
Result of `nixpkgs-review pr 158959` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>sqldef</li>
  </ul>
</details>

**Review by applePrincess — APPROVED** [review]

**Comment by applePrincess** [comment]
```
[nix-shell:~/.cache/nixpkgs-review/pr-158959]$ cat <<EOF | ./results/sqldef/bin/psqldef -U postgres -W ******** -h 172.17.0.2 -p 5432 test
create table items (
  item_id integer primary key,
  item_name varchar(100)
);
EOF
-- Apply --
create table items (
  item_id integer primary key,
  item_name varchar(100)
);


```

looks good to me.

**Comment by onny** [comment]
Please split commits into
```
sqldef: init at 0.12.7
maintainers: add kgtkr
```

**Comment by kgtkr** [comment]
@onny
Sorry for the delay, I missed the notification
I split the commit and reopened

**Comment by tomberek** [comment]
@kgtkr thanks for the contribution. Sorry it took so long to push everything through. I did a rebase of the commits to get the ordering of the maintainer-list right. After that everything worked out. Thanks again!

**Pull request merged**